prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -49,6 +49,17 @@ class FakeReport {
                     when {
                         element.nameContains("lab_name") -> "Any lab USA"
                         element.nameContains("facility_name") -> "Any facility USA"
+                        element.nameContains("patient_age_and_units") -> {
```

Comment:
Interesting.  Our very-large-hospital customer mentioned to us that some states object strongly to being sent an Age field, since its redundant with birthday.    Other states _require_ it.    Sigh.